nextcord/member.py

```diff
@@ -282,6 +284,7 @@ def __init__(
         self._timeout: Optional[datetime.datetime] = utils.parse_time(
             data.get("communication_disabled_until")
         )
+        self._flags = data.get("flags", 0)
```

Comment:
```suggestion
        self._flags: int = data.get("flags", 0)
```
like all other attributes